contrib/syslog-ng-debun

```diff
@@ -707,7 +708,7 @@ acquire_syslog_pids () {
 		sngpid=$( cat ${piddir}/syslog-ng.pid )
 	else
 		# Handle when fhs is "linux"-like
-		sngpid=$( cat ${piddir}/syslog-ng.pid )
+		sngpid=$( $pseao pid,args | grep "[s]yslog-ng " | head -1 | while read PID CMD; do echo $PID; done )
```

Comment:
Me feels using syslog-ng.pid file safer. Why did you need to use ps?

Reply:
@furiel I decided in favor of this approach due to the fact that I had to use this approach when dealing with the auto-detection of pids for the running WEC instances. At the same time, it is also very well possible that there may be multiple syslog-ng instances running in parallel (the same may be also true in case of wec), and the old pidfile approach only showed one syslog-ng process from the many ones possible.
The added benefit of this approach is that this way the pid detection is done in the very same way in case of both wec and syslog-ng.